docs/api/ref/Metadata.md

```diff
@@ -279,6 +279,9 @@ interface ImplementationMetadata {
      *  `fdc3.getCurrentChannel` and `fdc3.leaveCurrentChannel` are implemented by
      *  the Desktop Agent.*/
     "UserChannelMembershipAPIs": boolean;
+    /** Used to indicate whether the experimental Desktop Agent Bridging
+     *  feature is implemented by the Desktop Agent.*/
+    DesktopAgentBridging: boolean;
```

Comment:
should it be `desktopAgentBridging`? The properties are inconsistent in terms of whether they start with capitals are not. The usual convention is to not start with capitals in JS/TS I think.

There also seems to be inconsistency of the use of `readonly`...

Reply:
For reference this was added in
- #753

They are consistent within that object - but yes not quite to the convention. 
Removing the capitalization would (unfortunately) be a breaking change.
readonly is presumably not transitive (haven't confirmed that though) so should probably have been applied in 2.0.

This probably needs a separate issue and PR, I'd suggest it removes the quotes and applies _readonly_, but should **not** remove the capitalization. Sound right to you?